bigtable/snippets/garbagecollection/garbagecollection_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package garbagecollection
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"cloud.google.com/go/bigtable"
+	"context"
+	"log"
+	"os"
+)
+
+func TestGarbageCollection(t *testing.T) {
+	ctx := context.Background()
+	project := os.Getenv("GOLANG_SAMPLES_BIGTABLE_PROJECT")
+	instance := os.Getenv("GOLANG_SAMPLES_BIGTABLE_INSTANCE")
+	if project == "" || instance == "" {
+		t.Skip("Skipping bigtable integration test. Set GOLANG_SAMPLES_BIGTABLE_PROJECT " +
+			"and GOLANG_SAMPLES_BIGTABLE_INSTANCE.")
+	}
+	adminClient, err := bigtable.NewAdminClient(ctx, project, instance)
+
+	tableName := "gc-table"
+	if err = adminClient.DeleteTable(ctx, tableName); err != nil {
+		log.Printf("Could not delete table %s: %v", tableName, err)
```

Comment:
Use `t.Log`? We try to avoid printing anything in tests unless it's related to a failure. So, if this isn't a failure, it could just be ignored/not printed at all.